modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -585,6 +585,11 @@ public String toVarName(String name) {
             name = "_u";
         }
 
+        // numbers are not allowed at the beginning
+        if (name.matches("^\\d.*")) {
```

Comment:
I guess this is needed here, in addition to https://github.com/OpenAPITools/openapi-generator/blob/1280221a42ce29343977c34f0d032b00199ffe64/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java#L615
if the name is all uppercase and thus matches this early return:
https://github.com/OpenAPITools/openapi-generator/blob/1280221a42ce29343977c34f0d032b00199ffe64/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java#L594-L596